cfn-templates/ec2-instance-state-notification.yaml

```diff
@@ -0,0 +1,57 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: >-
+  Creates a CloudWatch Event to monitor the ec2 instance hosting the Palo Alto
+  and if any change in the instance status occurs a sns notification will email
+  the designated distro group.
+
+Parameters:
+  EmailGroup:
+      Description: >-
+        This is the email distro group that will be notified in the event
+        the ec2 instance, hosting the palo alto, goes down.
+      Type: String
+
+  KmsKeyId:
+      Description: >-
+        Kms key used to encrypt the SNS topic.
+      Type: String
+
+Resources:
+  Ec2StatusSNSTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      TopicName: "Ec2DowntimeNotification"
+      DisplayName: "Ec2DowntimeNotification"
+      KmsMasterKeyId: !Ref KmsKeyId
+
+
+  Ec2StatusSnsSubscription:
+    Type: AWS::SNS::Subscription
+    Properties:
+      Endpoint: !Ref EmailGroup
+      Protocol: email
+      TopicArn: !Ref Ec2StatusSNSTopic
+
+
+  CloudWatchEc2SnsStatus:
```

Comment:
Suggest consistency in the resource names.  Two of these three use `Sns`, and the third uses `SNS`

Reply:
addressed/fixed